sdk/servicebus/microsoft-azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/ManagementClient.java

```diff
@@ -3,22 +3,15 @@
 
 package com.microsoft.azure.servicebus.management;
 
-import com.microsoft.azure.servicebus.ClientSettings;
-import com.microsoft.azure.servicebus.Utils;
-import com.microsoft.azure.servicebus.primitives.AuthorizationFailedException;
-import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
-import com.microsoft.azure.servicebus.primitives.MessagingEntityAlreadyExistsException;
-import com.microsoft.azure.servicebus.primitives.MessagingEntityNotFoundException;
-import com.microsoft.azure.servicebus.primitives.QuotaExceededException;
-import com.microsoft.azure.servicebus.primitives.ServerBusyException;
-import com.microsoft.azure.servicebus.primitives.ServiceBusException;
-import com.microsoft.azure.servicebus.primitives.TimeoutException;
-import com.microsoft.azure.servicebus.primitives.Util;
+import com.microsoft.azure.servicebus.*;
```

Comment:
Star imports are not allowed. You may have to update IntelliJ to not change it to a star import.